Vagrantfile-quick

```diff
@@ -5,16 +5,26 @@
 # For building the "quick start" box file, the default Vagrantfile is used,
 # like on dev vms created with full saltstack provisioning.
 
-# On buildhost, following vagrant plugins must be installed:
-#  - vagrant-hostmanager
-#  - vagrant-vbguest
+require './helpers'
+require 'mkmf'
 
-# Helpers
-def colorize(text, color_code); "#{color_code}#{text}\033[0m"; end
-def red(text); colorize(text, "\033[31m"); end
-def yellow(text); colorize(text, "\033[33m"); end
-def green(text); colorize(text, "\033[32m"); end
-def bold(text); colorize(text, "\033[1;97m"); end
+unless find_executable 'git'
+  abort red "ERROR: git executable was found."
+end
+
+# Cleanup mkmf log
+File.delete('mkmf.log') if File.exists?('mkmf.log')
+
+required_plugins = ['vagrant-vbguest', 'vagrant-hostmanager']
```

Comment:
If you would like to optimise it, you can create an init function in helpers and just call it here and above.